client/src/app/account/services/account.service.ts

```diff
@@ -0,0 +1,60 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, Subject, tap } from 'rxjs';
+import { AccountInterface } from '../types/account.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AccountService {
+  accounts: AccountInterface[] = [];
+  accountsUpdated = new Subject<AccountInterface[]>();
+
+  constructor(private http: HttpClient) {}
+
+  addNewAccount(
+    userId: string | null,
+    title: string,
+    currency: string,
+    description: string
+  ) {
+    return this.http
+      .post<{ message: string; newAccount: AccountInterface }>(
+        'http://localhost:5000/accounts',
+        {
+          userId,
+          title,
+          currency,
+          description,
+        }
+      )
+      .pipe(
+        tap(() => {
+          this.accounts.push({
```

Comment:
In this case better to call `requestUserAccounts` from component to keep account list up to date, than leave this responsibility to `addNewAccount`.